coderd/database/queries/workspaceagentstats.sql

```diff
@@ -246,27 +246,17 @@ minute_buckets AS (
 latest_buckets AS (
 	SELECT DISTINCT ON (agent_id)
 		agent_id,
-		minute_bucket,
 		session_count_vscode,
+		session_count_ssh,
 		session_count_jetbrains,
-		session_count_reconnecting_pty,
-		session_count_ssh
+		session_count_reconnecting_pty
 	FROM
 		minute_buckets
 	ORDER BY
 		agent_id,
 		minute_bucket DESC
-), latest_agent_stats AS (
-	SELECT
```

Comment:
This subquery wasn't doing anything, since we've already selected distinct on `agent_id`